fuel-tx/src/tests/bytes.rs

```diff
@@ -733,8 +733,8 @@ fn script_input_coin_data_offset() {
                             .inputs_predicate_offset_at(offset)
                             .expect("Failed to fetch offset");
 
-                        assert_ne!(bytes::padded_len(&predicate), predicate.len());
-                        assert_eq!(bytes::padded_len(&predicate), len);
+                        assert_ne!(bytes::padded_len(&predicate), Some(predicate.len()));
```

Comment:
Why even include the `ne` case? There are many things that it's not equal to :)

Reply:
No idea. It was there already, and I just kept the old tests as-is.